apps/cli/src/handleListCliCommand.ts

```diff
@@ -22,26 +24,71 @@ export const handleListNamesCommand = async (printer: Printer) => {
 		onlyFiles: true,
 	});
 
-	const codemodNames = await Promise.allSettled(
+	const codemodObjects = await Promise.allSettled(
 		configFiles.map(async (cfg) => {
 			const configJson = await readFile(cfg, 'utf8');
 
 			const parsedConfig = v.safeParse(
-				v.object({ name: v.string() }),
+				v.object({
+					name: v.string(),
+					engine: v.string(),
+					owner: v.optional(v.string()),
+				}),
```

Comment:
i would make this schema a const somewhere else, since it's reused below L71

Reply:
Also, I'd make owner optional as well. We can handle this on our side, and it's better if we don't make it strict. Do you think it will always be present? What about download feature? What about `pnpm create` command in registry that creates a boilerplate? Not sure